grpcurl.yaml

```diff
@@ -27,6 +27,18 @@ pipeline:
       expected-sha256: 7f7a59f8a5ef8833d30a94e1c36ddb0d76bab1ae64cd5c8bcb87d42e877c3bca
       uri: https://github.com/fullstorydev/grpcurl/archive/refs/tags/v${{package.version}}.tar.gz
 
+  - runs: |
+      # Mitigate CVE-2022-27664, CVE-2021-33194, CVE-2021-31525, CVE-2022-41723
+      go mod edit -replace golang.org/x/net=golang.org/x/net@v0.7.0
```

Comment:
Related to [this discussion](https://github.com/chainguard-dev/enterprise-packages/pull/461#discussion_r1291669453)